README.md

```diff
@@ -28,6 +28,8 @@
 
 * **extensible**: via plugins
 
+* **fast**: on some older devices, it has been measured to have less than half the page-turn delay as the built in reading software.
```

Comment:
Less than half? Have they improved performance that much? I'd be inclined to say less than a tenth is still vastly understating it. :-P

Reply:
However, keep in mind much the same should apply to Plato and coolreader for example.

Reply:
> Less than half?

My average page turn latency went from 1.23s with Nickel to 0.58s with KOReader

Reply:
I suppose that's all you need to make it so that if you quickly tap next 20 times you're 20 pages further, vs. maybe 5 if you're lucky. I've never done any measurements.

Reply:
That's not my use case. The thing I care about is: I'm reading a book normally, and I go to turn the page, and I don't want it to take so much time that I consciously notice I am waiting for the page turn to finish. 

When it takes 1.23s to turn the page, I feel like I am being impeded every single time. When it's 0.58s, I don't really feel that way.

Reply:
In CRe, actual processing latency should be around 50ms (+100ms if not portraitl, the rest is e-ink ;).

For KePub, nickel should not be too far from that (say, 150ms), but, it suspends the device after 5s of inactivity, so, in practice, you have to eat that cost plus the cost of the system resume...

For plain ePub, RMSDK is slow as molasses, and you can fairly easily fall asleep between page turn ;p (> 1000ms).

Reply:
Keep in mind that an Aura HD's screen is also noticeably slower than more recent devices ;).